envVariables.js

```diff
@@ -61,127 +61,165 @@ let enviromentVariables = {
     "optional" : false,
     "default" : "ON"
   },
-  "CLOUD_STORAGE" : {
-    "message" : "Enable/Disable cloud services",
+  // "CLOUD_STORAGE" : {
```

Comment:
@Prajwal17Tunerlabs remove unused variables from the list

Reply:
removed unsued variables